### ISSUE TITLE
[TeleMelody] How to modify the tonality in the inference code?

### ISSUE THREAD

**Issue by elricwan** [microsoft/muzic#75]
Hi there, Thank you first for providing the great checkpoints so that I can play with some examples. During the experiments, I notice that I could modify the chord file to control the chord progression. And I also notice that in the paper, the tonality could also be controlled, so where could I change this value? 

Thank you.

**Comment by jzq2000** [comment]
Hi @elricwan . You can modify the ``mode`` variable ( ``MAJ`` or ``MIN`` ) in the [code](https://github.com/microsoft/muzic/blob/main/telemelody/inferrence/infer_en.py#L256) to control the tonality during inference.

**Comment by elricwan** [comment]
Hi @jzq2000 ,Thank you for the explanation! By the way, there are many more tonality exists like C, D, E, F, G, etc. Is there a specific reason that teleMelody trained on the C major and A minor tonality? 

Thank you.

**Comment by jzq2000** [comment]
@elricwan Yes. It is a common trick since transposition (other majors to C major / other minors to A minor) is trivial and rule-based. Thus, we need to consider one specific major and one specific minor, and normalize other tonalities to these two.
